entrypoint.sh

```diff
@@ -7,16 +7,31 @@ then
   export VAULT_TOKEN=$(echo $ENCRYPTED_VAULT_TOKEN | base64 --decode | aws kms decrypt --ciphertext-blob fileb:///dev/stdin --output text --query Plaintext --region $AWS_REGION | base64 --decode)
 fi
 
-if [ ${VAULT_TOKEN} ] && [ ${CONSUL_ADDR} ] && [ ${VAULT_ADDR} ]
+if [ ${CONSUL_ADDR} ]
 then
-  if consul-template -consul=$CONSUL_ADDR -template=/exports.ctmpl:/tmp/exports.sh -once -max-stale=0
+  if consul-template -consul=$CONSUL_ADDR -template=/consul-exports.ctmpl:/tmp/consul-exports.sh -once -max-stale=0
   then
-    source /tmp/exports.sh
+    source /tmp/consul-exports.sh
   else
-    echo "======== Consul or Vault are misbehaving. If you are seeing this in prod, Engineering Ops have been alerted. ========"
+    echo "======== Consul may be misbehaving. If you are seeing this in prod, Engineering Ops have been alerted. ========"
     exit 1
   fi
 else
-  echo "VAULT_TOKEN, VAULT_ADDR, or CONSUL_ADDR are not set skipping exports"
+  echo "CONSUL_ADDR are not set skipping Consul exports"
+fi
+exec "$@"
+
+
+if [ ${VAULT_TOKEN} ] && [ ${CONSUL_ADDR} ] && [ ${VAULT_ADDR} ] 
+then
+  if consul-template -consul=$CONSUL_ADDR -template=/vault-exports.ctmpl:/tmp/vault-exports.sh -once -max-stale=0
+  then
+    source /tmp/vault-exports.sh
```

Comment:
I think we will need to write two exports.sh files and source them both as well.